### DISCUSSION TITLE
16/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#155] [Daily Sprint]
Que hice ayer 
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer estuve en la clase de emprendedurismo 
No avance en nada porque estuve con activadas del colegio.
La organización en mí casa.

**Comment by rpgrca** [comment]
Ayer estuve en la clase de tecnología y desarrollo a la tarde, después me puse con la aplicación de Android haciendo que el login y el logout funcionen a través de la API del back-end. Hoy voy a estar en la clase de práctica profesionalizante aunque me hayan sacado del curso. Quiero ver si hoy puedo traer el catálogo del servidor y mostrarlo en la app.

**Comment by LeonardoEC** [comment]
Ayer trabaje todo el día y estube en clases.
Hoy trabajo y luego a seguir estudiando

**Comment by fmdcba** [comment]
Hoy estuve metiendo mano a selenium webdriver para probar la ui. Cree para un script para que navegue por los botones del nav tanto en full size como en el botón hamburguer. Para el login ingresa datos incorrectos así muestra las validaciones del form.
Algunas dificultades que me encontré fue que a la hora de probar el login con credenciales válidas al clickear "ingresar" no logro que me muestre la redirección al home. (Sigo leyendo doc y probando)
También tuve que importar una función "time" de python para meter un delay a la ejecucion de las pruebas y asi poder ver lo que pasa (Por ende por cada linea donde haga click o mande algun texto coloco debajo un `time.sleep(2)` y termina viendose muy repetitivo).  También quiero probar corriendo las pruebas en un headless e ir imprimiendo msjs en la consola

**Comment by LuciaL152** [comment]
Ayer estuve trabajando en otro proyecto y estuve en la clase de tecnología , y hoy estuve en la clase de emprendedurismo.
